app/models/user.rb

```diff
@@ -39,6 +39,7 @@ class User < ApplicationRecord
     state :new, initial: true
     state :transitioned
     state :rolled_over
+    state :seen_accredited_body_features
```

Comment:
Just to check, @TobyRet, are we leaving this here until we get the state machine functioning in Publish, and then rip it out as a separate piece of work?

Reply:
Yep that is correct